jsx/FilterableDataTable.js

```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react';
+
+import StaticDataTable from 'jsx/StaticDataTable';
+import FilterForm from 'jsx/FilterForm';
+
+class FilterableDataTable extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      filter: {},
+    };
+    this.updateFilter = this.updateFilter.bind(this);
+    this.clearFilter = this.clearFilter.bind(this);
+  }
+
+  updateFilter(filter) {
+    this.setState({filter});
+  }
+
+  clearFilter() {
+    this.updateFilter({});
+  }
+
+  render() {
+    return (
+      <div>
+        <FilterForm
+          name={this.props.name + '_filter'}
+          id={this.props.name + '_filter_form'}
+          columns={3}
+          filter={this.state.filter}
+          updateFilter={this.updateFilter}
+          clearFilter={this.clearFilter}
+        >
+          {this.props.formElements}
```

Comment:
would it make sense for this to be `this.props.children` instead?

Reply:
It could be, I don't see why not.

Reply:
I sent it as `this.props.children`